tests/functional/spa/basic.test.js

```diff
@@ -26,7 +26,7 @@ testDriver.test('capturing SPA interactions', supported, function (t, browser, r
       let interactionTree = (body && body.length ? body : querypack.decode(query.e))[0]
 
       t.equal(interactionTree.trigger, 'initialPageLoad', 'initial page load should be tracked with an interaction')
-      t.equal(interactionTree.children.length, 0, 'expect no child nodes')
+      t.ok(interactionTree.children[0].path.startsWith('/1/'), 'expect rum call child node')
```

Comment:
Do we know why this wasn't previously captured in a SPA page load but now it is?

Reply:
TIL, but page view event didn't use the `img` method before did it?  It looks like it injected a script elem onto the page. But that's also a valid reason for sure, since neither of those cases are "ajax" events.

Reply:
HAHA!! Yea I was confusing two different things. SO I did track that down. Our JSONP wrapper expects a parameter in the URL of `callback` or `cb`. Our rum call used `jsonp` as the query parameter. I honestly doubt our JSONP wrapping is effective.